### DISCUSSION TITLE
Los feeds de Blogger

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#40] [Announcements]
# Los feeds de Blogger

Los feeds de Blogger son archivos que contienen información sobre las entradas, comentarios, etiquetas y demás.

https://www.zkreations.com/2024/09/los-feeds-de-blogger.html

**Comment by ghost** [comment]
This was a very useful article. Thank you. By the way, RSS is working.

**Reply by danieIabel** [reply]
Alright, my friend, I'm glad everything is working now.

**Comment by binuball** [comment]
You can get maximum of 500 results using API v2 format.

**Reply by danieIabel** [reply]
Hi, friend. Wonderful contribution! I have updated the entry after verifying the information you provided. Thank you very much. A hug!

**Reply by binuball** [reply]
Update: we can't get more than 150 posts using API v2 format. I've just confirmed that. Is it just temporary error?

**Reply by danieIabel** [reply]
Hi friend, do you have a blog with many posts available for testing? I used to use Oloblogger, but now it redirects to Feedburner.

**Reply by binuball** [reply]
I don't have blog for testing. It's from my personal blog with 301 posts. But you can use any blog with more that 150 posts.
Also you can get blog id from comment iframe, ex: https://www.blogger.com/comment/frame/BLOG_ID?po=POST_ID

**Reply by danieIabel** [reply]
Hi again, my friend. I don't have a blog with many posts to run the test, but I will create one and verify the information you provided so I can update the entry as soon as possible.

**Reply by danieIabel** [reply]
I just checked with a blog from an acquaintance, and indeed, the API v2 is now limited to 150 entries. Personally, I don't think this is temporary, as Blogger has been optimizing resource usage over the past few years, so I would say this change is permanent.

Another thing I just noticed is that entries from blogs set to private can no longer be retrieved either, so I would say there's currently no difference in using any of the feeds, except, of course, for the V3 API.

Thank you so much for reporting this, my friend. I've already updated the article thanks to your help. Big hug!

**Comment by ghost** [comment]
Hi Daniel, I have a question for you. How do you think a personal newspaper website should be designed on Blogger?  [https://imagz.jagodesain.com](https://imagz.jagodesain.com) What do you think?  Or what are your thoughts about the newspaper website?

**Reply by danieIabel** [reply]
Good morning, my friend. I know Jagodesain, and their designs are excellent. I highly recommend them.

**Reply by ghost** [reply]
Thank you.

**Comment by ghost** [comment]
Hi Daniel, I hope you are well. I have a question for you. How did you make a text Twitter card for non-visual posts on Twitter?

**Reply by danieIabel** [reply]
Hi friend, I am using the Cloudinary API to generate these images combined with Blogger code. Do you think it would be a good idea to create a contribution about it?

**Reply by ghost** [reply]
Yes, I would be happy if you write an article about this.

**Comment by ghost** [comment]
Hi Daniel, if I ask you for a code, can you write an article for this? Can you write a code to print the page on the screen in Blogger? Thanks. By the way I sent question your email address.

**Reply by danieIabel** [reply]
Good morning, my friend. I apologize for the delayed response; the university is taking up almost all of my time. On top of that, my country is facing some energy generation problems, which leaves us without electricity for 14 hours every day. As soon as I get everything sorted out, I'll take the time to answer the questions you sent to my email.

**Reply by ghost** [reply]
No problem, friend, I hope things get better soon.
